.github/workflows/sdk-update-api-spec.yaml

```
@@ -0,0 +1,38 @@
+name: SDK Update API Spec
+
+on:
+  schedule:
+    - cron: '0 12 * * 3'
+  workflow_dispatch: {}
```

```suggestion
  workflow_dispatch:
```
This *should* also work.